### DISCUSSION TITLE
ДОБАВИТЬ В РАДИОПАК В GMME ВЕНДОР СМАРТГАННЕРА

### DISCUSSION THREAD

**Discussion by KarlFranko721** [Cosmic-Overlord/TerraGov-Marine-Corps#319] [Ideas]
На данный момент в вендорах смарта из уникального есть его ствол, КЛТД и медиум генерал поуч, который по сути генерал поуч по вместимости, на этом все, больше ничего нет. Радиопак же добавит немного уникальности и поможет с микро менеджментом патрон.

**Comment by fe-ed** [comment]
УРА НАСИЛИЕ НАД РО
